polkadot/runtime/parachains/src/paras_inherent/benchmarking.rs

```diff
@@ -124,13 +146,18 @@ benchmarks! {
 		// There is 1 backed,
 		assert_eq!(benchmark.backed_candidates.len(), 1);
 		// with `v` validity votes.
-		let votes = min(scheduler::Pallet::<T>::group_validators(GroupIndex::from(0)).unwrap().len(), v as usize);
+		let votes = min(
+			scheduler::Pallet::<T>::group_validators(GroupIndex::from(0)).unwrap().len(),
+			v as usize,
+		);
 		assert_eq!(benchmark.backed_candidates.get(0).unwrap().validity_votes().len(), votes);
 
 		benchmark.bitfields.clear();
 		benchmark.disputes.clear();
-	}: enter(RawOrigin::None, benchmark)
-	verify {
+
+		#[extrinsic_call]
+		enter(RawOrigin::None, benchmark);
+
 		let max_validators_per_core = BenchBuilder::<T>::fallback_max_validators_per_core();
```

Comment:
Check if this line is really useful somehow (I doubt it, in case write `_max_validators_per_core` for the unused variable). If it's not, delete it. The same below. See CI tests for such warnings.

Reply:
thx didnt catch it,  removed both unused variables, doesnt seem to be having any consequences on the tests